src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+		talkToGuardsAgainToTellThemYouAreReadyStep = new ConditionalStep(this, talkToGuardsAgainToTellThemYouAreReady);
+		talkToGuardsAgainToTellThemYouAreReadyStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+
+		/// 28 + 30
+		// TODO: what happens if you accuse the wrong person?
+		interrogateConstantiniusAgain = new NpcStep(this, NpcID.CONSTANTINIUS, new WorldPoint(1446, 2931, 2), "Interrogate Constantinius.");
+		interrogateConstantiniusAgain.addDialogStep("No.");
+		interrogateXocotlaAgain = new NpcStep(this, NpcID.XOCOTLA, new WorldPoint(1444, 2928, 2), "Interrogate Xocotla.");
+		interrogateXocotlaAgain.addDialogStep("No.");
+		interrogateCozyacAgain = new NpcStep(this, NpcID.COZYAC, new WorldPoint(1442, 2929, 2), "Interrogate Cozyac.");
+		interrogateCozyacAgain.addDialogStep("No.");
+		interrogatePavoAgain = new NpcStep(this, NpcID.PAVO, new WorldPoint(1442, 2931, 2), "Interrogate Pavo.");
+		interrogatePavoAgain.addDialogStep("No.");
+
+		accuseAdala = new NpcStep(this, NpcID.ADALA, new WorldPoint(1446, 2933, 2), "Accuse Adala of the crime, ready for an easy fight. You can use Protect from Melee to negate all damage.");
+		accuseAdala.addDialogStep("Accuse Adala.");
+
+		// TODO: helper steps to climb upstairs?
+		speakToSuspects = new ConditionalStep(this, accuseAdala);
+		speakToSuspects.addStep(not(inVilla), returnToButlerAndHeadInside);
+		speakToSuspects.addStep(not(interrogatedConstantiniusAgain), interrogateConstantiniusAgain);
+		speakToSuspects.addStep(not(interrogatedXocotlaAgain), interrogateXocotlaAgain);
+		speakToSuspects.addStep(not(interrogatedCozyacAgain), interrogateCozyacAgain);
+		speakToSuspects.addStep(not(interrogatedPavoAgain), interrogatePavoAgain);
+
+		/// 32
+		getAdalasConfession = new NpcStep(this, NpcID.ADALA_13821, new WorldPoint(1446, 2933, 2), "Talk to Adala to get her confession.");
+		// TODO: helper steps to climb upstairs?
+		getAdalasConfessionStep = new ConditionalStep(this, getAdalasConfession);
+		getAdalasConfessionStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+
+		/// 33
+		talkToGuardsAboutAdala = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1442, 2933, 2), "Talk to the guards about Adala.");
+		// TODO: helper step to climb upstairs?
+		talkToGuardsAboutAdalaStep = new ConditionalStep(this, talkToGuardsAboutAdala);
+		talkToGuardsAboutAdalaStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+
+		/// 34
+		// NOTE: The stairs don't highlight well. A tile step would be better, but tile steps don't work here.
+		var headDownFromTopFloor = new ObjectStep(this, ObjectID.STAIRCASE_54714, new WorldPoint(1445, 2939, 2), "Climb down the staircase.");
+		var headDownFromMiddleFloor = new ObjectStep(this, ObjectID.STAIRCASE_54714, new WorldPoint(1442, 2936, 1), "Climb down the staircase.");
+		var climbFirstLooseRocksToTheatre = new ObjectStep(this, ObjectID.LOOSE_ROCKS_54720, new WorldPoint(1469, 2918, 0), "Climb the Loose rocks on your way to the theatre.");
```

Comment:
Would be good to specify where the rocks are, like 'Climb the loose rocks south-east of the villa on your way to the theatre.'

Reply:
:+1: Fixed in [`8cff659` (#1774)](https://github.com/Zoinkwiz/quest-helper/pull/1774/commits/8cff659f0980474a41ee6e5db06c656dc5f7ab5e)